### PR TITLE
update to Laptop and Phone Security Policy

### PR DESCRIPTION
**Pull request by EThom73**
**What:**
Important updates to the Laptop and Phone security policy in line with ISO 27001
**Why**  
This covers our obligation to document and make available ot the team our policy around laptop and mobile device security.
**Notes:**
The links are to documents that don't exist/exist only in draft. They will be updated once the structure of the IMS file is established

### CODE REVIEW

File: ims/company/device_security.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Device Security
+
+This is general advice on device security. For the purposes of this section, "device" not only applies to the laptop Made Tech will issue you with when you join the company, but also any personal devices you use to access Made Tech systems. This includes, but is not limited to, phones, tablets, etc. 
+
+Please consult our BYOD policy and our AUP policy for more information (linked below).
```
Comment (lukemorton) [inline ims/company/device_security.md]: Should we link to BYOD and AUP here too?

File: ims/company/device_security.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+This is general advice on device security. For the purposes of this section, "device" not only applies to the laptop Made Tech will issue you with when you join the company, but also any personal devices you use to access Made Tech systems. This includes, but is not limited to, phones, tablets, etc. 
+
+Please consult our BYOD policy and our AUP policy for more information (linked below). 
+
+## Laptop Setup
+
+This section details policy around Made Tech issued devices. If you are using your own device at Made Tech please consult our BYOD policy (linked below)
+
+All Made Tech devices are set up with separate admin and user profiles, unless otherwise agreed with the relevant IMS governance team or leadership. For more information on this please see our Standard Builds policy (linked below).
```
Comment (lukemorton) [inline ims/company/device_security.md]: Let's just link here too.

Comment (lukemorton) [reply ims/company/device_security.md]: See "Reference-style links" in this markdown documentation for a way of making multiple links to the same content in a single document easier to manage: https://daringfireball.net/projects/markdown/syntax#link

Comment (EThom73) [reply ims/company/device_security.md]: That's a bit of a scary read for a non-tech person. 😬 
I will try and puzzle it out and make it work.

**Pull request merged**